build.py

```diff
@@ -216,7 +219,7 @@ def saveScriptAndMeta(script, ourDir, filename, oldDir=None):
 
 downloadUrl = distUrlBase and distUrlBase + '/total-conversion-build.user.js' or 'none'
 updateUrl = distUrlBase and distUrlBase + '/total-conversion-build.meta.js' or 'none'
-main = doReplacements(main, downloadUrl=downloadUrl, updateUrl=updateUrl)
+main = doReplacements(main, downloadUrl=downloadUrl, updateUrl=updateUrl, dateTimeVersion=getDateTimeVersion('main.js'))
```

Comment:
Instead of just `main.js` we should iterate over `code/*.js` to find newest file.